clean-and-build.sh

```
@@ -133,6 +133,9 @@ do
   done
 done
 
+./bin/ccd-add-role.sh "caseworker-sscs-panelmember PRIVATE"
```

@chrisdavecm I think the quotes here is causing the script to fail early